driver-portal/src/Controllers/test.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace MigrationMetrics.Controllers
```

Comment:
This namespace needs to be changed

Reply:
Adjusted the namespace and pushed the changes.